bazel/emscripten_toolchain/wasm_cc_binary.bzl

```diff
@@ -134,6 +123,10 @@ wasm_cc_binary = rule(
         "simd": attr.bool(
             default = False,
         ),
+        "outputs": attr.output_list(
+            allow_empty = False,
```

Comment:
I added a default outputs when the `outputs` attribute is empty. That does mean that #986 would still be broken (maybe?) when the outputs are empty.

Reply:
Ah I see. Unfortunately simply wrapping the rule in a macro for default outputs wouldn't work either since currently on the `main` branch the outputs depend on the `cc_target`'s name, not just the `wasm_cc_binary`'s name.

If the rule was wrapped in a macro and it's okay to break the default output paths not considering `cc_target`s name then that's great! Otherwise there could be _two_ internal rules that the macro chooses from.

I'm capable of doing either option. If you have a preference let me know and I'll make the necessary changes.

Reply:
I went ahead and created a macro that decies to use either a "legacy" rule or the new rule. I thought that might be better than parsing the `cc_target` label in the macro. https://github.com/emscripten-core/emsdk/pull/1047/commits/ce1cea7d8cbe2722503c945948afe59aa8a74059